src/DataTable.php

```diff
@@ -332,7 +334,12 @@ public function getColLabels(): array
             $value = __(Str::headline($key));
```

Comment:
Hol das lieber hier an der Stelle aus dem columnLabels, also $this->columnLabels[$key] ?? $key

Das spart das zweite Array Walk und das Array merge

Reply:
Aber was ist dann, wenn man nicht alle oder gar keine entries in `$columnLabels` hat? Dann bleiben diese Werte ja komplett leer, oder? Manche Labels können ja auch Default bleiben.

Reply:
Achso, mit `$this->columnLabels[$key] ?? $key` ist das ja abgedeckt. Passt, mach ich.